vhdbuilder/packer/install-dependencies.sh

```diff
@@ -427,6 +427,7 @@ PATCHED_HYPERKUBE_IMAGES="
 1.14.8_f0.0.4
 1.14.8-hotfix.20200326
 1.14.8-hotfix.20200408.1
```

Comment:
This was just added, do we need it?

Reply:
removed 
1.14.8-hotfix.20200408.1

Reply:
yes, this one was just added , and can be removed